README.md

```diff
@@ -31,8 +31,9 @@
   </a>
 </p>
 
-simplistika.css is a lightweight, responsive and modern CSS framework.
+simplistika.css is a single-file, lightweight, responsive and modern CSS framework.
 
+- Incredibly KISS.
```

Comment:
What does KISS mean?

Reply:
https://acronyms.thefreedictionary.com/KISS means "Simple" in a nutshell. It is mostly used for very bare stuff, such as directly copying a file and using it.

Reply:
How is straightforward to replace KISS? Not sure if KISS is common.

Reply:
...or just simple?

Reply:
Web devs get it, its common for them.

Reply:
ok